xml/operations-compute-gpu_passthrough.xml

```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="gpu-passthrough">
+  <title>GPU passthrough</title>
+  <para>
+    &productname; GPU passthrough functionality provides the nova instance direct access to the GPU device for a
+    better performance.
+  </para>
+  <para>
+    A lot of use cases are around combining OpenStack based Clouds with
+    a (sub-)set of compute hosts that have GPUs for pass-through
+    GPU computing. This section demonstrates the steps to pass
+    through a Nvidia GPU card supported by &productname;,
```

Comment:
I would remove the first sentence here... just have it say "This section demonstrates...." . The use cases bit is unnecessary

Reply:
Updated